core/src/main/java/hudson/tasks/BatchFile.java

```diff
@@ -40,10 +40,17 @@
  */
 public class BatchFile extends CommandInterpreter {
     @DataBoundConstructor
-    public BatchFile(String command) {
+    public BatchFile(String command, Integer unstableReturn) {
```

Comment:
Use of `@DataboundSetter` for new arguments would be preferable over adding new constructors.